publish/tests/test_version.py

```diff
@@ -30,6 +31,25 @@ def test_version_from_girder(dandiset_factory, mock_girder_client):
     assert version
 
 
+@pytest.mark.django_db
+def test_version_from_girder_no_metadata(dandiset_factory, mock_girder_client):
+    dandiset = dandiset_factory()
```

Comment:
I'm not sure it's sustainable to rely on the fact that the current `DandisetFactory` happens to have a `draft_folder_id` which renders it invalid when used with `Version.from_girder`. Could you at least add a code comment, explaining this fact?

Reply:
I added a code comment in https://github.com/dandi/dandi-publish/pull/63/commits/7117a04d5ec021fc59e39c25f3d6539e39fce5b6